artiq/dashboard/explorer.py

```diff
@@ -251,6 +251,14 @@ def scan_repository():
         scan_repository_action.triggered.connect(scan_repository)
         self.el.addAction(scan_repository_action)
 
+        rescan_ddb_action = QtWidgets.QAction("Re-scan device database on master",
```

Comment:
It's always "on master" and never specified in the dashboard elsewhere. The dashboard is always a client.